Certificate revocation documentation (#2986)
kyma-project/kyma: docs/application-connector/docs/08-09-revoke-cert.md

---

> title: Revoke the certificate

Reviewer: ```suggestion
title: Revoke the client certificate
```

type: Tutorials
---

> This tutorial will show you how to revoke certificate, making it unable to be renewed.

Reviewer: ```suggestion
You can revoke a client certificate generated for your Application. Revocation prevents a certificate from being renewed. A revoked certificate, however, continues to be valid until it expires.
```

> To revoke certificate make a call to ``certificates/revocations`` endpoint:

Reviewer: ```suggestion
To revoke a client certificate, send a request to the `certificates/revocations` endpoint. Pass the certificate you want to revoke and a key that matches this certificate in the call. Run:
```

    
```bash
curl -X POST https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/revocations --cert {CERT_TO_REVOKE} --key {CERT_TO_REVOKE_KEY} -k 
```
    

> After successful call, the certificate is revoked and cannot be renewed.

Reviewer: Remove this line, brings no value.

Author: Done

> If you have access to Kyma cluster, you can also revoke a certificate by making a call with certificates sha256 sum to internal ``certificates/revocations`` endpoint:

Reviewer: ```suggestion
If you have access to the Kyma cluster, you can also revoke a certificate by sending the SHA256 sum of the certificate to the internal `certificates/revocations` endpoint. Run:
```
